core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -60,26 +61,14 @@ class Invoker(
     instance: Int,
     activationFeed: ActorRef,
     verbosity: LogLevel = InfoLevel,
-    runningInContainer: Boolean = true)(implicit actorSystem: ActorSystem)
-    extends MessageHandler(s"invoker$instance")
-    with Logging {
+    runningInContainer: Boolean = true)(implicit actorSystem: ActorSystem, logging: Logging)
+    extends MessageHandler(s"invoker$instance") {
 
     private implicit val executionContext: ExecutionContext = actorSystem.dispatcher
-    private implicit val emitter: PrintStreamEmitter = this
 
     /** This generates completion messages back to the controller */
     val producer = new KafkaProducerConnector(config.kafkaHost, executionContext)
 
-    override def setVerbosity(level: LogLevel) = {
-        super.setVerbosity(level)
-        pool.setVerbosity(level)
-        entityStore.setVerbosity(level)
-        authStore.setVerbosity(level)
-        activationStore.setVerbosity(level)
-        producer.setVerbosity(level)
-        checkRuncAccess()
```

Comment:
Never mind. This call has been removed in commit 31f08b840927f184980a4f71080fa25468b4da40